public-api/src/main/resources/private-workbench.yaml

```diff
@@ -0,0 +1,93 @@
+# This file is for endpoints we do NOT intend to generate client libraries for
```

Comment:
I would put a disclaimer about the intended lifespan on this swagger file and trim some of the fat in these comments. If we wanted something longer term we'd probably want to avoid duplicating our own swagger yaml

Reply:
Yeah, I'll throw some more comments in there.